portalnet/src/utp_controller.rs

```diff
@@ -51,6 +51,60 @@ impl UtpController {
         }
     }
 
+    pub fn cid(&self, peer: UtpEnr, is_initiator: bool) -> ConnectionId<UtpEnr> {
+        self.utp_socket.cid(peer, is_initiator)
+    }
+
+    // `try_acquire_owned()` isn't blocking and will instantly return with
+    // `Some(TryAcquireError::NoPermits)` error if there isn't a permit avaliable
+    pub fn get_outbound_semaphore(&self) -> Option<OwnedSemaphorePermit> {
+        match self
+            .outbound_utp_transfer_semaphore
+            .clone()
+            .try_acquire_owned()
+        {
+            Ok(permit) => Some(permit),
+            Err(_) => None,
+        }
+    }
+
+    // `try_acquire_owned()` isn't blocking and will instantly return with
+    // `Some(TryAcquireError::NoPermits)` error if there isn't a permit avaliable
```

Comment:
```suggestion
    /// `try_acquire_owned()` isn't blocking and will instantly return with
    /// `Some(TryAcquireError::NoPermits)` error if there isn't a permit avaliable
```
These are doc strings hence should use /// right?